app/views/layouts/home.html.erb

```diff
@@ -10,12 +10,13 @@
     </div>
   </div>
   <div class="quick-buttons large-5 columns">
-    <ul class="box list">
+    <ul class="box list text-center">
         <li><a data-tooltip title="TimeEdit" href="https://se.timeedit.net/web/chalmers/db1/public/ri1Q7.html" target="_blank" class="has-tip tip-top"><%= fa_icon 'table', text: t('.schema') %></a></li>
         <li><a data-tooltip title="<%= t '.book_hubben_desc' %>" href="https://bookit.chalmers.it/bookings/new" target="_blank" class="has-tip tip-top"><%= fa_icon 'home', text: t('.booking') %></a></li>
         <li><a data-tooltip title="<%= t '.book_chalmers_desc' %>" href="https://se.timeedit.net/web/chalmers/db1/b1/" target="_blank" class="has-tip tip-top"><%= fa_icon 'users', text: t('.group_room') %></a></li>
         <li><a data-tooltip title="<%= t '.print_desc' %>" href="<%= new_print_path %>" class="has-tip tip-top"><%= fa_icon 'print', text: t('.print') %></a></li>
         <li><a data-tooltip title="<%= t '.charge_card_desc' %>" href="http://kårkort.se" target="_blank" class="has-tip tip-top"><%= fa_icon 'credit-card', text: t('.charge_card') %></a></li>
+        <li><a data-tooltip title="<%= t '.slack_desc' %>" href="https://slack.chalmers.it/signup" target="_blank" class="has-tip tip-top"><%= fa_icon 'slack', text: t('.slack') %></a></li>
```

Comment:
Did you check how it looks on mobile? That layout broke easily by my experience...